### DISCUSSION TITLE
'Light Up' Design Philosophy

### DISCUSSION THREAD

**Discussion by fireprone** [christoph-codes/bulb#11] [Show and tell]
@christoph-codes: I'm experimenting with a new style where the positive space of the page is using a dark color to represent a 'shadow'. The negative space, on the other hand, is a white-adjacent color to represent light. 

I added some hover styling on the `landing-page` branch if you'd like to take a look at it in action. Let me know what you think!

**Comment by christoph-codes** [comment]
> @christoph-codes: I'm experimenting with a new style where the positive space of the page is using a dark color to represent a 'shadow'. The negative space, on the other hand, is a white-adjacent color to represent light. 
> 
> 
> 
> I added some hover styling on the `landing-page` branch if you'd like to take a look at it in action. Let me know what you think!

Sounds interesting but I will definitely check this out and provide feedback.

**Reply by christoph-codes** [reply]
@fireprone where else do you think we can use this. I realized I never responded to this after seeing it in action!
